docs/root/intro/version_history.rst

```diff
@@ -1,6 +1,65 @@
 Version history
 ---------------
 
+1.5.0
+=====
+
+* access log: added fields for :ref:`UPSTREAM_LOCAL_ADDRESS and DOWNSTREAM_ADDRESS
+  <config_access_log_format>`.
+* admin: added :ref:`JSON output <operations_admin_interface_stats>` for stats admin endpoint.
+* admin: added basic :ref:`Prometheus output <operations_admin_interface_stats>` for stats admin
+  endpoint. Histograms are not currently output.
+* config: the :ref:`v2 API <config_overview_v2>` is now considered production ready.
+* config: added ::option:`--v2-config-only` CLI flag.
+* cors: added :ref:`CORS filter <config_http_filters_cors>`.
+* health check: added :ref:`x-envoy-immediate-health-check-fail
+  <config_http_filters_router_x-envoy-immediate-health-check-fail>` header support.
+* health check: added :ref:`reuse_connection <envoy_api_field_HealthCheck.reuse_connection>` option.
+* http: added :ref:`per-listener stats <config_http_conn_man_stats_per_listener>`.
+* http: end-to-end HTTP flow control is now complete across both connections, streams, and filters.
+* load balancer: added :ref:`subset load balancer <arch_overview_load_balancer_subsets>`.
+* load balancer: added ring size and hash :ref:`configuration options
+  <envoy_api_msg_Cluster.RingHashLbConfig>`.
+* log: added the ability to optionally log to a file instead of stderr via the
+  ::option:`--log-path` option.
+* listeners: added :ref:`drain_type <envoy_api_field_Listener.drain_type>` option.
+* lua: added experimental :ref:`Lua filter <config_http_filters_lua>`.
+* mongo filter: added :ref:`fault injection <config_network_filters_mongo_proxy_fault_injection>`.
+* mongo filter: added :ref:`"drain close" <arch_overview_draining>` support.
+* outlier detection: added :ref:`HTTP gateway failure type <arch_overview_outlier_detection>`.
+* redis: the :ref:`redis proxy filter <config_network_filters_redis_proxy>` is now considered
+  production ready.
+* redis: added :ref:`"drain close" <arch_overview_draining>` functionality.
+* router: added :ref:`x-envoy-overloaded <config_http_filters_router_x-envoy-overloaded>` support.
+* router: added :ref:`regex <envoy_api_field_RouteMatch.regex>` route matching.
```

Comment:
this went into 1.4 itself right? or is it just this release?

Reply:
The commit that added this support was in this release cycle. It's possible the option got added but not implemented?